web/crux/src/app/node/node.container.ws.gateway.ts

```diff
@@ -75,13 +80,23 @@ export default class NodeContainerWebSocketGateway {
     )
   }
 
+  @SubscribeMessage(WS_TYPE_SET_CONTAINER_LOG_TAKE)
+  setContainerLogTake(
+    @NodeId() nodeId: string,
+    @SocketMessage(NodeContainerLogQueryValidationPipe) message: SetContainerLogTakeMessage,
+  ): void {
+    this.service.setContainerLogTake(nodeId, message)
```

Comment:
If multiple users are watching the same container log with different take lengths, the latest take will be used right?